--- a/XML_RPC/XML/RPC/Server.php	2017-12-30 12:30:06.000000000 +0000
+++ b/XML_RPC/XML/RPC/Server.php	2017-12-30 12:32:12.000000000 +0000
@@ -330,6 +330,10 @@
     {
         global $HTTP_RAW_POST_DATA;

+        if (empty($HTTP_RAW_POST_DATA)) {
+            $HTTP_RAW_POST_DATA = file_get_contents("php://input");
+        }
+
         if ($debug) {
             $this->debug = 1;
         } else {
